pytest-pixl/src/pytest_pixl/helpers.py

```diff
@@ -0,0 +1,92 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Testing utilities"""
+from __future__ import annotations
+
+import logging
+import subprocess
+from time import sleep
+from typing import TYPE_CHECKING, Callable, Optional
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+
+def run_subprocess(
```

Comment:
as we've got this, can we replace any other calls across tests with this function?

Reply:
Have done all of them except one, which is using `text=True` which the wrapper doesn't support.